### DISCUSSION TITLE
Correct way of the ESP on the PCB

### DISCUSSION THREAD

**Discussion by Liriel-666** [pmgducati/Blue-Retro-AIO-Units#73] [General]
What way is the esp too fit on the main pcb?
like here on the Picture from the linked Blog?
https://amaiorano.io/assets/images/blueretro-aio/receiver/IMG_9223.jpg

Because when i look on the pins of the main PCb and the pins of my esp itshould the other way.

sry that i ask but i got today the pcbs

**Comment by pmgducati** [comment]
There is a comment under the picture in the blog:
(Note that the ESP32 board is actually inserted the wrong way in the pic above, as this was only temporary to solder in the sockets).

Let me know if you are still having issues.  I reached out to the blog author to remove/update the picture

**Comment by Liriel-666** [comment]
i cleared it with the blog author days ago because that i thaught i closed the discussion
